mock-client/tokens.ts

```diff
@@ -33,14 +36,32 @@ export class TokensController extends BaseController {
   })
   async getTokens(
     @Paginated() _paginationParams?: PaginatedParams,
-  ): Promise<PaginatedTokenResponse> {
+  ): Promise<OcpiResponse<TokenDTO>> {
+    console.log(_paginationParams);
+
+    const fail = false;
+
+    if (fail) {
+      return buildGenericServerErrorResponse();
+    }
+
+    const token = generateMockOcpiResponse(TokenDTO);
+
+    token.uid = (2 + (_paginationParams?.offset ?? 0)).toString();
+    token.country_code = 'US';
+    token.party_id = 'MSP';
+    token.type = TokenType.APP_USER;
+
     const response = buildOcpiPaginatedResponse(
       OcpiResponseStatusCode.GenericSuccessCode,
       10,
```

Comment:
even though this is mock, still may be good to reuse defaults, I think we should have somewhere